make/00_mod.mk

```diff
@@ -32,12 +32,9 @@ deploy_name := istio-csr
 deploy_namespace := cert-manager
 
 helm_chart_source_dir := deploy/charts/istio-csr
-helm_chart_name := cert-manager-istio-csr
+helm_chart_image_name := quay.io/jetstack/charts/cert-manager-istio-csr
```

Comment:
It's not really an "image" is it? Might have been more intuitive to use something with `oci_` prefix, for consistency with the existing vars.

```
$ git grep oci_  make/0?_mod.mk
make/00_mod.mk:oci_manager_base_image_flavor := static
make/00_mod.mk:oci_manager_image_name := quay.io/jetstack/cert-manager-istio-csr
make/00_mod.mk:oci_manager_image_tag := $(VERSION)
make/00_mod.mk:oci_manager_image_name_development := cert-manager.local/cert-manager-istio-csr
make/00_mod.mk:oci_platforms := linux/amd64,linux/arm/v7,linux/arm64,linux/ppc64le,linux/s390x
make/00_mod.mk: '( .image.repository = "$(oci_manager_image_name)" ) | \
make/00_mod.mk: ( .image.tag = "$(oci_manager_image_tag)" )' \
make/02_mod.mk: @echo "RELEASE_OCI_MANAGER_IMAGE=$(oci_manager_image_name)" >> "$(GITHUB_OUTPUT)"
make/02_mod.mk: @echo "RELEASE_OCI_MANAGER_TAG=$(oci_manager_image_tag)" >> "$(GITHUB_OUTPUT)"
```

Reply:
This will require a change in https://github.com/cert-manager/makefile-modules.
I use `helm_chart_image_name` since it roughly matches the name `oci_manager_image_name`.